### ISSUE TITLE
Property names that are reserved words in Java generate funky getter and setter methods

### ISSUE THREAD

**Issue by Helmsdown** [swagger-api/swagger-codegen#525]
DefaultCodegen.getterAndSetterCapitalize() delegates toVarName() to handle removing invalid characters from a property name (e.g. removing a "-" from a propertyName). 

The JavaClientCodegen.toVarName(), in addition to removing invalid characters, escapes reserved keywords with an underscore.

This results in Java getter and setter methods like the following

```yaml
default:
  type: String
  description: the description
```

```java
private String _default = null;

public String get_default() {
  return _default;
}

public String set_default(){
  return _default;
}
```

Perhaps we need a third function in DefaultCodeGen that only removes invalid characters (and does not escape reserved words). This would be used by getterAndSetterCapitalize(), instead of toVarName() which does both.

**Comment by fehguy** [comment]
Yes, the two should probably be separate.

**Comment by wing328** [comment]
@Helmsdown this should be addressed in #535 

I did a test before and setter, getter for "return" is as follows:

```
  /**
   **/
  @ApiModelProperty(required = false, value = "")
  @JsonProperty("return")
  public String getReturn() {
    return _return;
  }
  public void setReturn(String _return) {
    this._return = _return;
  }
```

**Comment by Helmsdown** [comment]
Great! If thats the case this issue can be closed.
